examples/sample-app/README.md

```diff
@@ -170,6 +191,8 @@ the present working directory is the same directory as this README.
       you'll see the page update as you deploy objects into the project
       and run builds.
 
+	**VAGRANT USERS**:  web console will not work correctly.
```

Comment:
And I would state this as:

    **VAGRANT_USERS**: Per above, the web console will not be available.